### ISSUE TITLE
Make Table header sticky for when rows don't fit in the window

### ISSUE THREAD

**Issue by basanta078** [iTwin/iTwinUI-react#72]
#### Feature
Table header goes out of the view when scrolling down when rows don't fit in the window and user needs to scroll down
As you can see in the screenshot below, header is still shown when user scrolls down few rows 

#### Examples
![image](https://user-images.githubusercontent.com/2604235/118832332-87508500-b88e-11eb-908d-c53e8cad3d03.png)

**Comment by mayank99** [comment]
What happens if you just set a fixed height (and `overflow: hidden`) on the table?

**Comment by mayank99** [comment]
@basanta078 Check out this code sandbox. https://codesandbox.io/s/itwinui-react-minimal-example-forked-9lbun?file=/src/styles.scss

I set `height: 100%` on table, so I didn't have to set height on body.

**Comment by basanta078** [comment]
> @basanta078 Check out this code sandbox. https://codesandbox.io/s/itwinui-react-minimal-example-forked-9lbun?file=/src/styles.scss
> 
> I set `height: 100%` on table, so I didn't have to set height on body.

Yes, thank you,  setting overflow-y: auto for .iui-tables-body did the trick
I had to do some extra since I want scrolling on x axis if the width is scrollable too.
Also I am using sass so I had to use :global to not mess with the body class name

Here is what I ended up doing:

`import styles from 'mytable.module.scss'`

```jsx
<div className={styles.container}>
  <Table
    className={styles.table}
    {...props}
  />
</dv>
```
```sass
.table {
  height: 100%;
  :global(.iui-tables-body) {
    overflow-y: auto;
    overflow-x: hidden;
  }
}

.container {
  overflow-y: hidden;
  overflow-x: scroll;
  flex-grow: 1;
}
```
Here vertical scrollbar is controlled by the table body
Where as horizontal scrollbar is controlled by the table container

![image](https://user-images.githubusercontent.com/2604235/119016565-154c6e80-b968-11eb-946f-0ce8dea90e03.png)

**Comment by mayank99** [comment]
Just a heads up, we've released 1.8.0 which includes some big internal changes to Table.

The table body class name has been changed to `iui-table-body` but also, you won't need to set overflow there anymore because it's included in our css now. All you need to do is provide height.

```diff
 .table {
   height: 100%;
-  :global(.iui-tables-body) {
-    overflow-y: auto;
-    overflow-x: hidden;
   }
 }
```

**Comment by basanta078** [comment]
> Just a heads up, we've released 1.8.0 which includes some big internal changes to Table.
> 
> The table body class name has been changed to `iui-table-body` but also, you won't need to set overflow there anymore because it's included in our css now. All you need to do is provide height.
> 
> ```diff
>  .table {
>    height: 100%;
> -  :global(.iui-tables-body) {
> -    overflow-y: auto;
> -    overflow-x: hidden;
>    }
>  }
> ```

This messes up the header ui which is not scrollable while the body is scrollable:
![image](https://user-images.githubusercontent.com/2604235/125354882-55a9e680-e332-11eb-8bf1-324a74297f69.png)


After 1.8.0 
![image](https://user-images.githubusercontent.com/2604235/125354951-69ede380-e332-11eb-886f-abccd322dfe5.png)
